### ISSUE TITLE
Field name change since previous releases

### ISSUE THREAD

**Issue by CMIP-IPO** [CMIP-Data-Request/CMIP7_DReq_Software#20]
@rigoudyg Just to note the field names have changed slightly:
1. 'MIPs' in the Variable Group table has changed to 'Of interest to MIPs'.
2.  'Variable Groups' field in the MIPs table has changed to 'Variable Groups of interest'

**Comment by rigoudyg** [comment]
Thanks for the reminder.
I have done a pull request which should deal with that.
I don't have the database yet to check that it is ok.

Just to be sure:
>     2. 'Variable Groups' field in the MIPs table has changed to 'Variable Groups of interest'
It was 'Variable Group', no?
So it becomes 'Variable Group of interest' or 'Variable Groups of interest'?
I put all versions in the changes so it should not be a problem.

**Comment by CMIP-IPO** [comment]
Yes you are right - it was 'Variable Group'. It will become 'Variable Groups of interest'

**Comment by CMIP-IPO** [comment]
@rigoudyg following a meeting yesterday, it was decided that the field 'Time Slices' should be renamed to 'Time Subsets'. I am going through the Airtable and updating in there. The following places need renaming in the three base export:
1. In Opportunities base, table 'Time Slice' renamed to 'Time Subset'
2. In 'Time Subset' table fields 'SliceLen' and 'sliceLenUnit' renamed to 'SubsetLen' and 'subsetLenUnit' respectively
3. Opportunities base, 'Opportunities' table, field 'Time Slice is renamed to 'Time Subset'

Single base export:
1. Table 'Time Slice' renamed to 'Time Subset'
2. In 'Time Subset' table fields 'SliceLen' and 'sliceLenUnit' renamed to 'SubsetLen' and 'subsetLenUnit' respectively
3. 'Opportunities' table, field 'Time Slice is renamed to 'Time Subset'

I won't change the previous static version bases (so v1.0, v1.0beta, and v1.0 alpha will still refer to Time Slices)

**Comment by CMIP-IPO** [comment]
@rigoudyg not sure if this also requires additional updates, but there will be a new column in the Variables table called 'ESM-BCV 1.4', and a new Table called 'ESM-BCV 1.4'. In the three base export, this table will sit in the Variable base.

**Comment by rigoudyg** [comment]
@CMIP-IPO does it replace the ESM-BCV 1.3 or is it an other one?

**Comment by CMIP-IPO** [comment]
@rigoudyg I can do either:
a) replace the existing ESM-BCV 1.3 table/fields with the new v1.4
b) keep the ESM-BCV 1.3 table/fields and create a new table/fields for v1.4

What is easier for you to manage?

**Comment by rigoudyg** [comment]
@CMIP-IPO 
with (a), it should work without any change.
with (b), I will have to change a few things but it's not a problem.
Do what is the easier for you and I will adapt the code if needed.

Maybe, having a content branch test with an extract of the export in which the changes could be done could help having a complete workflow test. I did it by hand for the sofware tests but it could help us testing the sofware before the content release. Maybe to discuss on Thursday.

**Comment by CMIP-IPO** [comment]
Ok great. Thinking about it, it will have to be new fields for the v1.4 so lets go with b) - I will create a new table and fiels for the BCVs v1.4. However, I will remove:
1) The 'ESM-BCV 1.3' table from both the single and three base export
2) The 'ESM-BCV 1.3' field from the Variables table in both the single and three base export.

Lets chat about your idea on Thursday.

**Comment by CMIP-IPO** [comment]
Name changes for next release to highlight - 
1. In the 3-base export, in Opportunities base, Opportunity table, the field 'Ensemble Size' has changed to 'Minimum Ensemble Size'
2. In the 3-base export, in Opportunities base, Opportunity table, the field 'Notes' has changed to 'Technical Notes'
3. In the 1-base export, in Opportunity table, field 'Notes' has changed to 'Technical Notes'

**Comment by bethdingley** [comment]
### More field name changes to flag

| Export | Table name | Old field name | New field name |
|--------|------------|----------------|-----------------|
| Both | Variables table | 'Modelling Realm'  | 'Modelling Realm - Primary' |
| Both | Variables table | 'Compound name' | 'CMIP6 Compound Name' |
| Both | Variables table | 'Table' | 'CMIP6 Table (legacy)' |
| Both | Modeling Realm table | 'Variables' | 'Variables - primary realm' |
| Release | Data Request Themes table | 'UID 2' | 'UID' |
| Raw | CF Standard Name table | 'Physical Parameters 2' | 'Physical Parameters' |
| Release | CMIP7 Frequency table | 'CMIP6 Frequency 2' | 'CMIP6 Frequency (legacy)' |
| Raw | Coordinate or Dimension table | 'Requested Bounds]' | 'Requested Bounds' |
| Raw | Time subset table | 'uid' | 'UID' |
| Release | Variables table | 'Contitional' | 'Conditional' |


### Table name changes
| Export | Old table name | New table name | 
|--------|------------|----------------|
| Both | 'Table Identifiers' | 'CMIP6 Table Identifiers (legacy)' |

### New fields
| Export | Table name | New field name | 
|--------|------------|----------------|
| Both | Variables table | 'Modelling Realm - Secondary' |
| Both | Variables table | 'CMIP7 Compound Name' |
| Both | Modeling Realm table | 'Variables - secondary realm' | 
| Both | Variables table | 'Branded Variable Name' |
| Both | Opportunities table | 'Image' |
| Raw | Coordinate or Dimension table | 'Notes' |
| Raw | Coordinate or Dimension table | 'Variables' |
| Raw | Coordinate or Dimension table | 'Size' |




### Fields removed
| Export | Table name | Removed field name | 
|--------|------------|----------------|
| Release | Variables table | 'Proposed CF Standard Name (for new Physical Parameters)' |
| Release | Variables table | 'Structure Label' |
| Both | Variables table | 'Structure Title' |
| Both | Spatial Shape table | 'Structure' |
| Both | Temporal Shape table | 'Structure' |
| Release | Coordinates and Dimensions table | 'Structure' |
| Raw (3-base) | Coordinate or Dimension table | 'Structure' |
| Both | Cell Methods table | 'Structures' |
| Both | Cell Measures table | 'Structures' |
| Release | Spatial Shape table | 'Hor Label DD' |
| Release | Spatial Shape table | 'Vertical Label DD' |
| Release | Spatial Shape table | 'Vertical Label MM' |
| Release | Temporal Shape table | 'Brand' |
| Both | Cell Methods table | 'Brand ID' |
| Both | Cell Methods table | 'Brand tag' |
| Release | ESM-BCV 1.4 table | 'Structure Title (from Variables)' |
| Raw (3-base) | ESM-BCV 1.4 table | 'Structure Title (from CMOR Variables)' |
| Both | CMIP6 Table (legacy) table | 'Table' | 'Status' |
| Raw (3-base) | Data Request Themes table | 'Experiment group' |
| Raw (3-base) | Experiment Group table | 'Themes to alert' |
| Release | MIPs table | 'MIP Feedback' |
| Release | Physical Parameters table | 'Proposed CF Standard Name' |
| Release | Physical Parameters table | 'CF Proposal Github Issue' |
| Release | Physical Parameters table | 'CF Standard Name Proposal Accepted' |
| Release | Physical Parameters table | 'Does a CF standard name exist for this parameter?' |
| Release | Physical Parameters table | 'Name validation' |



### Tables removed
| Export | Removed table name | 
|--------|------------|
| Both | 'Structure' table |

**Comment by bethdingley** [comment]
New fields for v1.2.2

| Export | Table name | New field name |
-- | -- | --
Both | Coordinates and Dimensions table | 'Variables with extra dimensions'
Release | Variables table | 'Extra dimensions'
Both | Physical parameters table | 'variableRootDD'

**Comment by bethdingley** [comment]
Another new field:

Export | Table name | New field name
-- | -- | --
Both | Variables table | 'Region'
